### PR TITLE
Compact latitude/longitude button

### LINKED ISSUE #11391: Optimize the Waypoint selection
### Tell us your idea!

Let the waypoint selection using the compact form of a single Lat/Lon-Button instead of 2 buttons like introduced in #11372.
Use "Coordinate" instead of "Coordinates" when referencing a single coordinate.

related to #11372
related to #11390

### What solution would you suggest?

_No response_

### Is there an alternative?

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by bekuno**
## Description

Follow up to the design optimization for the coordinate selection presented in https://github.com/cgeo/cgeo/issues/11354#issuecomment-890254665 .

- replace the 2 coordinate buttons by a single double line button for the search activity and the editwaypoint activity
- Defining a slightly larger font for the coordinates for better readability
- fix some string problems
  - "geo code" to "geocode"
  - "cachename" to "cache name"
  - "Keywords" to "Keyword" in search headline
  - some upper case/lower case optimizations

There is no change to the problem coordinates vs. coordinate.
The opinions were different in #11390.

## Related issues

Releated to #11354
Releated to #11390

Fixes #11391

### CODE REVIEW

**Comment by bekuno** [comment]
Some impressions.

Search dialog:
Screenshot_20210927 Search 1  Screenshot_20210927 Search 2

Waypoint dialog:
Screenshot_20210927 waypoint 1 Screenshot_20210927 waypoint 2

File: main/res/values/strings.xml [hunk 1]
```diff
@@ -393,7 +394,7 @@
     <string name="caches_sort_favorites">Favorites</string>
     <string name="caches_sort_favorites_ratio">Favorites [%]</string>
     <string name="caches_sort_name">Name</string>
-    <string name="caches_sort_geocode">Geo Code</string>
+    <string name="caches_sort_geocode">Geocode</string>
```
Comment (Lineflyer) [inline main/res/values/strings.xml:397]: Is it one word in english? I am not sure.

File: main/res/values/strings.xml [hunk 2]
```diff
@@ -1335,10 +1336,10 @@
     <string name="cache_status_disabled">Disabled</string>
     <string name="cache_status_active">Active</string>
     <string name="cache_status_premium">Premium Members only</string>
-    <string name="cache_geocode">Geo code</string>
+    <string name="cache_geocode">Geocode</string>
```
Comment (Lineflyer) [inline main/res/values/strings.xml:1339]: see above

File: main/res/values/dimens.xml [hunk 3]
```diff
@@ -42,6 +42,7 @@
 
     <!-- buttons -->
     <dimen name="textSize_buttonsPrimary">@dimen/textSize_detailsSecondary</dimen>
+    <dimen name="textSize_buttonsDouble">@dimen/textSize_detailsPrimary</dimen>
```
Comment (Lineflyer) [inline main/res/values/dimens.xml:45]: Is this the same text size as the coordinates currently have in the single button?

Comment (bekuno) [reply main/res/values/dimens.xml:45]: I increased the size a little bit. Before it was the same as the button text (As "SEARCH BY COORDINATES"). But it was bad to read the coordinate without additional space between the coordinate parts. And we have now more space :-)

File: main/res/values/strings.xml (hunk 1 above)
Comment (bekuno) [reply main/res/values/strings.xml:397]: Yes, it is.
See our conversation in the first PR at https://github.com/cgeo/cgeo/pull/11372#discussion_r680344380.

Comment (Lineflyer) [reply main/res/values/strings.xml:397]: Thanks, now I remember :)

File: main/res/values/dimens.xml (hunk 3 above)
Comment (Lineflyer) [reply main/res/values/dimens.xml:45]: Sounds good.

**Review by Lineflyer — APPROVED** [review]

**Comment by bekuno** [comment]
retest this please

**Pull request merged**